src/lib/ruby-generator/control.js

```diff
@@ -46,8 +46,8 @@ export default function (Blockly) {
     };
 
     Blockly.Ruby.control_stop = function (block) {
-        const target = block.getFieldValue('STOP_OPTION') || null;
-        return `stop("${target}")\n`;
+        const target = Blockly.Ruby.quote_(block.getFieldValue('STOP_OPTION')) || null;
```

Comment:
`|| null`も、 `Blockly.Ruby.quote_` の引数の中に入れてくださいませ。かなり手間ですみませんが、他の箇所もお願いします。

```suggestion
        const target = Blockly.Ruby.quote_(block.getFieldValue('STOP_OPTION') || null);
```

Reply:
作業方針としてはいい感じです。

Reply:
ざっとみたところ、 `valueToCode` のところはこのままで良さそうです。修正が必要そうなのは `getFieldValue` のところだけかな。